wolfcrypt/src/random.c

```diff
@@ -1556,18 +1556,14 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 #elif defined(HAVE_RTP_SYS) || defined(EBSNET)
 
 #include "rtprand.h"   /* rtp_rand () */
-#include "rtptime.h"   /* rtp_get_system_msec() */
 
 
 int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 {
-    int i;
-    rtp_srand(rtp_get_system_msec());
```

Comment:
Why remove all "seeding"?  Shouldn't the first one (not in the loop) be retained?

Reply:
I checked with Sarah about that. Their stack seeds the rng before this is called. I'm double-checking with her about adding the first one back in.